plio/settings.py

```diff
@@ -86,6 +87,7 @@
     "rest_framework_social_oauth2",
     "storages",
     "silk",
+    "etl",
```

Comment:
why does it need to be in the tenant app given that we are storing the `schema` in the table inside the table in the `public` schema itself?